mockkube/src/test/java/io/strimzi/test/io/strimzi/test/mockkube/MockKubeTest.java

```diff
@@ -256,7 +256,7 @@ public void podNameScopedCreateListGetDelete(Class<RT> cls,
         gotResource = mixedOp.apply(client).withName(pod.getMetadata().getName()).get();
         assertThat(gotResource, is(nullValue()));
 
-        assertThat(mixedOp.apply(client).withName(pod.getMetadata().getName()).delete(), is(false));
+        assertThat(mixedOp.apply(client).withName(pod.getMetadata().getName()).cascading(true).delete(), is(false));
```

Comment:
I don't think the MockKube properly supports cascading deletion (i.e. I suspect it behaves the same for a cascading as a non-cascading delete), but if the tests pass I guess it doesn't matter.

Reply:
Yes, the test pass.